PACKAGING.md

```diff
@@ -118,6 +118,14 @@ relevant to package maintainers:
   version, glibc, etc. Run `zig targets` to a get a full list of available
   targets.
 
+- `-Demit-terminfo`: Install the Ghostty terminfo source file. Defaults to
+  `false` except on Windows where it defaults to `true`. Except for Windows,
+  this file is not required for proper operation as compiled versions of the
+  terminfo file are installed.
+
+- `-Demit-termcap`: Install the Ghostty termcap file. This may be useful for
```

Comment:
No need to document these in PACKAGING.md. We only document the most relevant configs. It's unlikely a packager will ever care here since our defaults are so sane. The goal of PACKAGING.md is to only give the most relevant actionable info so a packager succeeds, not to be a reference.

Reply:
Removed.